cron.go

```diff
@@ -166,13 +171,29 @@ func (c *Cron) Schedule(schedule Schedule, cmd Job) EntryID {
 		Job:        cmd,
 	}
 	if !c.running {
-		c.entries = append(c.entries, entry)
+		c.addNewEntry(entry)
 	} else {
 		c.add <- entry
 	}
 	return entry.ID
 }
 
+func (c *Cron) UpdateSchedule(id EntryID, spec string) error {
```

Comment:
I think we may need to offer a way to Update using a Schedule instead of just a spec string, similar to how you can add a job using either.

Perhaps named Update and UpdateSchedule ?

Reply:
Makes sense. I have added in the support for the same.